src/hash.hpp

```diff
@@ -20,7 +20,8 @@
 
 #include "global.hpp"
 #include "exceptions.hpp"
-#include "bcrypt/bcrypt.h"
+
+#define BCRYPT_HASHSIZE	(64)
```

Comment:
Please use a `static const` variable for this instead of a `#define`.

Reply:
Actually that was just a copy from libbcrypt as I eliminated it.

We need a buffer which is at least large enough to hold the result. That size is both hard-coded and implicitly coded in crypt_blowfish.

So both methods are poor choices because they imply it's a configurable factor; which it is not.

I'll just use the value (64) at the appropriate point.

Reply:
Nah. Used static const int.